### PR TITLE
Add some missing fields to fi_tostr function.

### PR DESCRIPTION
**Pull request by pmmccorm**
Add some notes about naming and format as well. I don't know what to do with
fi_domain_attr.domain , so I just print its value.

Signed-off-by: pmmccorm <patrick.m.mccormick@intel.com>

### CODE REVIEW

File: src/fi_tostr.c [hunk 1]
```diff
@@ -379,6 +421,9 @@ static void fi_tostr_domain_attr(char *buf, const struct fi_domain_attr *attr,
 	}
 
 	strcatf(buf, "%sfi_domain_attr:\n", prefix);
+
+	strcatf(buf, "%s%sdomain: %x\n", prefix, TAB, attr->domain);
```
Comment (goodell) [inline src/fi_tostr.c]: no `0x` prefix here?

**Comment by goodell** [comment]
Other than that small nit, the changes look good to me.

On my system this commit passes my round trip parsing test of:
```sh
$ fi_info | ruby -ryaml -e 'print YAML.dump YAML.load_stream(ARGF)'
```

File: src/fi_tostr.c (hunk 1 above)
Comment (pmmccorm) [inline src/fi_tostr.c]: Maybe? Not sure what we should do with that struct. I'll update the patch.

**Comment by pmmccorm** [comment]
Updated to prefix 0x. Passes the in-tree parser as well:

```
$ fi_info | scripts/parseyaml.py -v
```

**Pull request merged**